ajax/libs/gridifier/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "gridifier",
+  "version": "2.0.3",
+  "filename": "gridifier.min.js",
+  "description": "Async responsive HTML grids",
+  "homepage": "http://gridifier.io/",
+  "keywords": [
+    "grids",
+    "async-grids",
+    "grid-layouts"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nix23/gridifier"
+  },
+  "license": "CC BY-NC-SA 4.0",
```

Comment:
We should follow the SPDX format. You can take a look at [license-list.json](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L79) under `tools` directory.